src/utils.rs

```diff
@@ -6,6 +8,27 @@ pub fn get_project_dirs() -> Option<ProjectDirs> {
     ProjectDirs::from("com", "ccgauche", "ytermusic")
 }
 
+/// Locate the headers.txt file:
+pub fn locate_headers_file() -> Option<PathBuf> {
+
+    // Locate the headers.txt file:
+    let header_paths: [Option<PathBuf>; 2] = [
```

Comment:
You can skip the `Option<PathBuf>` and just use `PathBuf` using an unwrap or ? on the from_str

Reply:
@ccgauche I've taken a stab at changing that - assuming that I understood what you meant.

Reply:
@ccgauche

Reply:
@ccgauche please re-review this pr.